docs/tutorials/aqt/getting_started.ipynb

```diff
@@ -0,0 +1,268 @@
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/Cirq/blob/master/docs/tutorials/aqt/getting_started.ipynb\"><img src=\"https://www.tensorflow.org/images/colab_logo_32px.png\" />Run in Google Colab</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a target=\"_blank\" href=\"https://github.com/quantumlib/Cirq/blob/master/docs/tutorials/aqt/getting_started.ipynb\"><img src=\"https://www.tensorflow.org/images/GitHub-Mark-32px.png\" />View source on GitHub</a>\n",
+    "  </td>\n",
+    "  <td>\n",
+    "    <a href=\"https://storage.googleapis.com/tensorflow_docs/Cirq/docs/tutorials/aqt/getting_started.ipynb\"><img src=\"https://www.tensorflow.org/images/download_logo_32px.png\" />Download notebook</a>\n",
+    "  </td>\n",
+    "</table>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "b42eeeef4398"
+   },
+   "source": [
+    "[AQT](https://www.aqt.eu) supports Cirq as a third party software development kit and offers access to various quantum computing devices and simulators in the backend. Login to the [AQT Gateway Portal](https://gateway-portal.aqt.eu) to get a list of available devices.\n",
+    "\n",
+    "After the Cirq installation has finished successfully, you are ready to use different backends by the use of a token and the corresponding backend URL like in the following getting started tutorial.\n",
+    "\n",
+    "## Use your AQT credentials"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "id": "4e87b1af4806"
+   },
+   "outputs": [],
+   "source": [
+    "import cirq\n",
+    "from cirq.aqt.aqt_device import get_aqt_device\n",
+    "from cirq.aqt.aqt_sampler import AQTSampler\n",
+    "access_token = 'MY_TOKEN'"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "63a64281ca4e"
+   },
+   "source": [
+    "Where `MY_TOKEN` is your access token for a specific AQT device. You need to subscribe to an AQT backend at the [AQT Gateway Portal](https://gateway-portal.aqt.eu) and retreive the access token. Then you can access the AQT device by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "id": "4fbc40903531"
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "0───1\n"
+     ]
+    }
+   ],
+   "source": [
+    "device, qubits = get_aqt_device(2)\n",
+    "print(device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "576763bf5e42"
+   },
+   "source": [
+    "## Sample a quantum circuit\n",
+    "\n",
+    "You can then use that device in Cirq. For example, preparing the Bell state\n",
+    "\n",
+    "\\\\[ |\\psi\\rangle=\\frac{1}{\\sqrt{2}}(|00\\rangle-\\mathrm{i}|11\\rangle) \\\\]\n",
```

Comment:
This needs to be in `$$` signs to render correctly.

Reply:
> I'm happy to add 1 and 2 if you're okay with me pushing to your branch.

sure, go ahead, you can push to my branch

Reply:
changed to `$$`